package_manifest.yaml

```diff
@@ -640,6 +640,7 @@ foreman_plugin_packages_tier2:
     rubygem-foreman_webhooks: {}
     rubygem-foreman_vmwareannotations: {}
     rubygem-foreman_wreckingball: {}
+    rubygem-hammer_cli_foreman_resource_quota: {}
```

Comment:
This should be in `hammer_plugin_packages`

Reply:
Thanks @evgeni , changed. I was fiddling around with the add_gem_package script for a while :upside_down_face: